index.html

```diff
@@ -80,5 +80,15 @@ <h3>Improve how we navigate</h3>
         </div>
       </div>
     </section>
+    <section>
+      <h2>People say</h2>
+      <q class="text-center"
+        >Oly is a true consummate professional and a seriously black-belt
+        product designer. I had the absolute pleasure of managing Oly for over a
+        year and I can wholeheartedly say that his departure has been a huge
+        loss to our organistaion...</q
+      ><br />
```

Comment:
Avoid `br` as much as possible. It adds extra markup. Instead, use CSS to provide spacing between elements.